modules/swift/pages/troubleshooting-logs.adoc

```diff
@@ -137,14 +137,15 @@ include::swift:example$code_snippets/SampleCodeTest.swift[tags="file-logging", i
 
 ====
 
-<.> Set the log file directory
-<.> Here we change the max rotation count from the default (1) to 5. +
-*Note* this means six files may exist at any one time; the five rotated log files, plus the active log file
-<.> Here we set the maximum size (bytes) for our log file
-<.> Here we select the binary log format (included for reference only as this is the default)
-<.> Here we increase the log output level from the default (_warnings_) to _info_ -- see: https://docs.couchbase.com/mobile/{major}.{minor}.{maintenance-ios}{empty}/couchbase-lite-swift/Classes/FileLogger.html#/s:18CouchbaseLiteSwift10FileLoggerC5levelAA8LogLevelOvp[log.file.level: LogLevel]
-+
-*Note* that the use of https://docs.couchbase.com/mobile/{major}.{minor}.{maintenance-ios}{empty}/couchbase-lite-swift/Classes/Database.html#/s:18CouchbaseLiteSwift8DatabaseC11setLogLevel_6domainyAA0fG0O_AA0F6DomainOtFZ[Database.setLogLevel()] is now deprecated.
+<.> Set the log file directory.
+<.> Set the logging format to binary by disabling text logging. 
```

Comment:
it's a bit hard to see as you've done driveby fixes at the same time, but looks like you've corrected the order of callouts at the same time?

Reply:
If i get you correctly, yes I corrected the order of the callouts as the original texts did not have this, also did a driveby fix of the admonition.